### PR TITLE
Added gamepad controls and demo video to README

### LINKED ISSUE #50: Add documentation for supported actions
The initial set of actions is more or less complete, and it would be ideal to have some documentation providing the details of actions that the project supports.

### PR DESCRIPTION
**Pull request by dmahajan980**
Resolves #50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +58,51 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
...
+| `Button 11: Right Thumbstick Button` | Open a new window | - | `false` |
+| `Button 12: D-Pad Up Button` | Switch to the previous browser window | - | - |
+| `Button 13: D-Pad Down Button` | Switch to the next browser window | - | - |
+| `Button 14: D-Pad Left Button` | Switch to the previous browser tab | - | - |
+| `Button 15: D-Pad Right Button` | Switch to the next browser tab | - | - |
+| `Left Thumbstick Horizontal Direction` | Scroll horizontally | `1` | `false` |
+| `Left Thumbstick Vertical Direction` | Scroll vertically | `1` | `false` |
+| `Right Thumbstick Horizontal Direction` | History navigation | - | `false` |
+| `Right Thumbstick Vertical Direction` | Focus on the previous/next element | `2.5` | `false` |
+
```
Comment (the-t-in-rtf) [inline README.md:99]: It would be good here to briefly mention here the actions that aren't bound to a button in the default configuration. A separate table or bullet list is fine.  The unconfigured actions should also be linked to their section in the documentation.

File: README.md [hunk 2]
```diff
@@ -58,3 +58,51 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
...
+Although the gamepad inputs are reconfigurable, the extension provides a default configuration for each gamepad input.
+The default action for each gamepad input is given below.<br>
+(_Please refer to the above diagram for gamepad inputs_)
+
+| Gamepad Input | Default Action | Speed Factor | Invert Action / Open a new tab or window in background |
+| :---: | :--- | :---: | :---: |
+| `Button 0` | Click | - | - |
+| `Button 1` | None | - | - |
+| `Button 2` | History back button | - | - |
```
Comment (the-t-in-rtf) [inline README.md:81]: It would be helpful to link each action to its section in the documentation.

File: README.md [hunk 3]
```diff
@@ -58,3 +58,51 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
...
+   <img src="src/images/gamepad.svg">
+   <i>
+      <p align="center">
+         Image Source: <a href="https://tinyurl.com/y2wvtldg">W3C Gamepad API Documentation</a>
+      </p>
+   </i>
+</p>
+
+Although the gamepad inputs are reconfigurable, the extension provides a default configuration for each gamepad input.
+The default action for each gamepad input is given below.<br>
```
Comment (the-t-in-rtf) [inline README.md:74]: "given below" is a litttle funny, I'd suggest "as follows" instead.

File: README.md (hunk 2 above)
Comment (dmahajan980) [reply README.md:81]: Hi @the-t-in-rtf, thanks for the feedback. I'm not sure whether I understood this correctly, but do you mean that we should link the actions to the navigation invokers?

Comment (the-t-in-rtf) [reply README.md:81]: You spelled out elsewhere what each action does.  Because you're now in a slightly different context, it'd be good to at least mention it once, something like "For more information on each action, read the related docs", linked to the appropriate page.

Comment (dmahajan980) [reply README.md:81]: @the-t-in-rtf as far as I could recall, the only place we described what each action does is in the corresponding invoker definitions. So maybe we should add the list of actions here, with a brief description. We can also add a reference to the invoker's definition along if you allow.

**Comment by dmahajan980** [comment]
@the-t-in-rtf I've added a separate list of actions currently supported by the extension with their description and invoker documentation links. I thought adding the list on top would make it explicit rather than specifying the non-default actions separately. But kindly let me know if you want this to be in the same format you talked about, and I'll update it.

File: README.md (hunk 2 above)
Comment (the-t-in-rtf) [reply README.md:121]: That sounds like a good idea.

File: README.md [hunk 4]
```diff
@@ -59,6 +59,46 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
 _**Note**: The new tabs opened using the gamepad navigator will use [Google](https://www.google.com/) as the homepage and
 override the default new tab page. However, it can be configured to open any other website instead of Google._
 
+## Supported Actions
+
+The Gamepad Navigator currently supports the following actions for buttons and triggers:
+
+| Action | Description | Invoker Documentation |
+| :--- | :--- | :--- |
+| **Click** | Perform a click on the focused element when the gamepad button or trigger is pressed. | [click](docs/components/inputMapper.base.md#inputmapperbaseclickvalue) |
```
Comment (the-t-in-rtf) [inline README.md:68]: "Perform a click" can be simplified to just "Click".

Comment (the-t-in-rtf) [reply README.md:68]: Also, for consistency and brevity, you can leave out "when the gamepad button or trigger is pressed".

File: README.md [hunk 5]
```diff
@@ -59,6 +59,46 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
...
+| **Switch to the next browser tab** | Switch from the current tab to the next tab in the active browser window. | [goToNextTab](docs/components/messageListener.md#messagelistenergotonexttab) |
+| **Open a new browser window** | Opens a new window. | [openNewWindow](docs/components/messageListener.md#messagelisteneropennewwindowactive-homepageurl) |
+| **Close current browser window** | Close the currently active browser window. | [closeCurrentWindow](docs/components/messageListener.md#messagelistenerclosecurrentwindow) |
+| **Switch to the previous browser window** | Switch from the currently active browser window to the previous browser window. | [goToPreviousWindow](docs/components/messageListener.md#messagelistenergotopreviouswindow) |
+| **Switch to the next browser window** | Switch from the currently active browser window to the next browser window. | [goToNextWindow](docs/components/messageListener.md#messagelistenergotonextwindow) |
+| **History back button** | Navigate to the previous page in history. | [previousPageInHistory](docs/components/inputMapper.base.md#inputmapperbasepreviouspageinhistoryvalue) |
```
Comment (the-t-in-rtf) [inline README.md:85]: Now that I see them all together, I would reword these two history action  names.  Every other name clearly describes the action without referring to a "button", hotkey, or other equivalent.  It's a little difficult because a good name might end up being nearly as long as (and repeating much of) the description.  I'd suggest maybe losing "button" and rewording to "Back (history)" and "Next (history)".

File: README.md [hunk 6]
```diff
@@ -59,6 +59,46 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
...
+| **History next button** | Navigate to the next page in history. | [nextPageInHistory](docs/components/inputMapper.base.md#inputmapperbasenextpageinhistoryvalue) |
+| **Maximize the current browser window** | Maximize the current browser window. | [maximizeWindow](docs/components/messageListener.md#messagelistenermaximizewindowleft) |
+| **Restore the size of current browser window** | Restore the size of the currently active browser window. | [restoreWindowSize](docs/components/messageListener.md#messagelistenerrestorewindowsizeleft) |
+| **Re-open the last closed tab or window** | Reopen the last closed browser tab or window. | [reopenTabOrWindow](docs/components/messageListener.md#messagelistenerreopentaborwindow) |
```
Comment (the-t-in-rtf) [inline README.md:89]: I think "Reopen" without the hyphen would be a little better.

**Comment by dmahajan980** [comment]
@the-t-in-rtf I've updated the pull. Kindly review it again.

Thanks
